docs/modules/ROOT/pages/tutorial/deploy.adoc

```diff
@@ -61,10 +61,6 @@ image::tutorial-deploy-upgrade-wizard.png[Deploy block upgrade wizard]
 +
 image::tutorial-deploy-end-wizard.png[Deploy block end wizard]
 
-. Copy the wallet address of your relayer through the icon, and send some Goerli ETH to it. Although this relayer will send the deploy transaction, you will execute the upgrade transaction through your wallet, so leave some Goerli ETH in your wallet.
```

Comment:
removing this because because we are assuming auto funding for testnet relayers?

Reply:
Yes, but I recognize that may not always be possible. What are your thoughts?

Reply:
not possible if chain is not goerli or no funds available -- should be fine for now.